custom_typings/yeoman-environment.d.ts

```diff
@@ -4,7 +4,7 @@ declare module 'yeoman-environment' {
     registerStub(generator: any, namespace: string);
     run(target: string, options: any, done: Function);
     getGeneratorsMeta(): YeomanEnvironment.GeneratorMeta[];
-    lookup(callback: () => void);
+    lookup(callback: (error?: Error) => void);
```

Comment:
That's crazy, I don't think I've ever seen a reasonably-supported module return strings, numbers, etc. in a `error/err` callback argument.